packages/odyssey-react-mui/src/properties/translations/odyssey-react-mui_fr.properties

```
@@ -0,0 +1,7 @@
+field.error.screen.reader.text = Erreur
```

Should this be `screenreader.text`?

```suggestion
field.error.screenreader.text = Erreur
```

---

Great idea